fairmotion/ops/conversions.py

```diff
@@ -181,6 +181,16 @@ def R2Q(R):
     )
 
 
+def R2R6(R):
+    return R[..., 0:2]
+
+
+def R62R(R6):
+    R3 = np.cross(R6[..., 0], R6[..., 1])
+    R = np.concatenate((R6, np.expand_dims(R3, axis=-1)), axis=-1)
+    return R
+
+
```

Comment:
Although I used the terminology R6 for the simplicity, R2R6 and R62R might not be ideal names... Let's find how other people named it...

Reply:
Changed the name to R6D to avoid consecutive digits in previous method name `R62R`